### ISSUE TITLE
Default rotate drawable cropped while rotating

### ISSUE THREAD

**Issue by cprcrack** [naver/android-pull-to-refresh#17]
Look carefully at the screenshot to see the end of the arrows being cropped while the view is rotating. This happens because the image is not square and thus gets cropped while rotating. It could be fixed adding some padding to the png so that it is totally square.

Also, how to provide my own drawable to fix this temporarily? I have read https://github.com/nhnopensource/android-pull-to-refresh/wiki/Customization but I don't know how to override RotateLoadingLayout, I'm using the library as an AAR package. How can I change it externally, ie without having the source code?

![screenshot_2014-06-26-19-27-46](https://cloud.githubusercontent.com/assets/650925/3402080/999fb1f8-fd57-11e3-9728-0b0995c19fb4.png)

**Comment by ncoolz** [comment]
I'm sorry for late reply. This issue can be resolved by using custom RotateLoadingLayout externally (as you said) or using some other custom arrow drawable (which is already padded).

But, This issue is a bug, so I will fix it later.
I recommend you to use custom loading layout until the bug is fixed. The instruction is described below,

### 1. Create new loading layout class overriding RotateLoadingLayout

```
package yourpackage;

....

public class NewRotateLoadingLayout extends RotateLoadingLayout {

    public NewRotateLoadingLayout(Context context, Mode mode,
            Orientation scrollDirection, TypedArray attrs) {
        super(context, mode, scrollDirection, attrs);
    }

    @Override
    public void onLoadingDrawableSet(Drawable imageDrawable) {
        super.onLoadingDrawableSet(imageDrawable);
        int padding = 10;
        // Add padding to header image view   
        mHeaderImage.setPadding(padding, padding, padding, padding);
    }

}
```

#### 2. Create new "pulltorefresh.xml" file in /assets directory and fill it like below,

```
<?xml version="1.0" encoding="UTF-8"?>
<PullToRefresh>
    <LoadingLayouts>
     <layout name="new_rotate">yourpackage.NewRotateLoadingLayout</layout>
    </LoadingLayouts>
</PullToRefresh>
```

The value in <layout> element must be the class path of new loading layout class. "name" attribute can be set anything.

Some sample "pulltorefresh.xml" file is [here](https://github.com/naver/android-pull-to-refresh/blob/master/sample/assets/pulltorefresh.xml).

#### 3. In layout xml, set `ptrAnimationStyle` of PullToRefreshListView as below,

```
    <com.handmark.pulltorefresh.library.PullToRefreshListView
        ...
        ptr:ptrAnimationStyle="new_rotate"
        />
```

The value of ptrAnimationStyle above is "new_rotate". This value was defined as `<layout name="new_rotate">` in assets/pulltorefresh.xml.

A sample layout xml is [here](https://github.com/naver/android-pull-to-refresh/blob/master/sample/res/layout/activity_ptr_custom_loadinglayout.xml).

#### 4. Done.

**Comment by cprcrack** [comment]
Hey ncoolz, was this fixed in the last commits?

**Comment by ncoolz** [comment]
Fixed on 3.2.3 version.
